ext/odbc/tests/odbc_commit_001.phpt

```diff
@@ -0,0 +1,31 @@
+--TEST--
+odbc_commit(): Basic test for odbc_commit()
+--EXTENSIONS--
+odbc
+--SKIPIF--
+<?php include 'skipif.inc'; ?>
+--FILE--
+<?php
+
+include 'config.inc';
+
+$conn = odbc_connect($dsn, $user, $pass);
+odbc_exec($conn, 'CREATE TABLE commit_table (test INT)');
+
+odbc_autocommit($conn, false);
+odbc_exec($conn, 'INSERT INTO commit_table VALUES(1)');
+var_dump(odbc_commit($conn));
```

Comment:
You have to use a new connection to select the table, because select via the same connection will return the row (as epected). However, it looks like I cannot open a new connection while the other is open, as the process times out 🤔